config/jobs/kubernetes/sig-k8s-infra/trusted/releng/releng-trusted.yaml

```
@@ -145,6 +146,37 @@ periodics:
       - org: kubernetes
         slug: release-managers
 
+# This job is a canary job to test promoting the image promoter before
+# rolling changes out to production instances
+- interval: 1h
+  cluster: k8s-infra-prow-build-trusted
+  max_concurrency: 
```

I was mislead by the presubmit output, but the actual error seems to be that it does not like this line:

```suggestion
  max_concurrency:
```

from
```
config/jobs/kubernetes/sig-k8s-infra/trusted/releng/releng-trusted.yaml
  153:19    error    trailing spaces  (trailing-spaces)
```

---

Good catch. Fixed!